### PR TITLE
clarify that a real phone number is needed for testing

### PR DESCRIPTION
**Pull request by afeld**
I tried entering a fake phone number and got locked out; didn't understand that it was actually going to text/call! Still not entirely clear to me what the difference between the sandbox and production are in this regard; may want to strike the previous sentence, or further clarify. Thanks!

### CODE REVIEW

**Review by NatashaJIbrahim — COMMENTED** [review]
@afeld Thank you for this! I agree this should be made more clear. My understanding is that in Production we check that the phone is associated with the address of the user during Identity Proofing. However, in Sandbox this check doesn't happen since it depends on a third party service. This check also doesn't happen for basic authentication. 

I'm sorry you were locked out of your account. If it's any consolation, the Sandbox and Production accounts are discrete. One trick we use here when testing (that you might already be familiar with) is to append the date to the end of our email address for example, "my-email@example.com" becomes "my-email+06152022@example.com". They'll be treated as the same address by most email services but as different addresses by Login. This can allow you to have multiple testing accounts in the Login Sandbox environment. 

I'll look over this one more time in the morning.

**Comment by julialeague** [comment]
You can also get away with using a fake phone number, as long as you're ok with logging in with the generated backup codes for that user. Understandable if you don't! That could be clarified somewhere.

File: _pages/testing.md [hunk 1]
```diff
@@ -151,7 +151,7 @@ To simulate a failure, enter a social security number that does not start with 
 
 ### Phone number verification
```
Comment (Richalonso45) [inline _pages/testing.md:152]: 3523176827

Comment (Richalonso45) [inline _pages/testing.md:152]: 593635325

File: _pages/testing.md [hunk 2]
```diff
@@ -151,7 +151,7 @@ To simulate a failure, enter a social security number that does not start with 
...
 
-Login.gov collects a phone number during the proofing process. In a live production environment, Login.gov checks that this phone number is associated with the applicant. You can use any phone number for testing purposes in the sandbox environment other than the following:
+Login.gov collects a phone number during the proofing process. In a live production environment, Login.gov checks that this phone number is associated with the applicant. You can use any real phone number for testing purposes in the sandbox environment other than the following:
```
Comment (Richalonso45) [inline _pages/testing.md:154]: 3523176827

File: _pages/testing.md (hunk 1 above)
Comment (Richalonso45) [reply _pages/testing.md:152]: 593635325

**Review by Richalonso45 — APPROVED** [review]

File: _pages/testing.md (hunk 2 above)
Comment (Richalonso45) [reply _pages/testing.md:154]: 3523176827

**Comment by orenyk** [comment]
This will be resolved in a pending update to this page.

**Pull request closed without merging**